pkg/storage/sqlstorage/log.go

```diff
@@ -91,7 +91,6 @@ func (s *Store) AppendLog(ctx context.Context, logs ...core.Log) error {
 func (s *Store) lastLog(ctx context.Context, exec executor) (*core.Log, error) {
 	var (
 		l    core.Log
-		ts   sql.NullString
```

Comment:
why is this removed ? shouldnt it have been replaced by a NullTime ?

Reply:
The Scan() method is called directly with a pointer on the date : 
```err := rows.Scan(&l.ID, &l.Type, &l.Hash, &l.Date, &data)```
No need to handle NULL values as the date is required.